IHP/AutoRefresh.hs

```diff
@@ -153,16 +152,15 @@ registerNotificationTrigger touchedTablesVar autoRefreshServer = do
 
     let subscriptionRequired = touchedTables |> filter (\table -> subscribedTables |> Set.notMember table)
     modifyIORef autoRefreshServer (\server -> server { subscribedTables = get #subscribedTables server <> Set.fromList subscriptionRequired })
-    forEach subscriptionRequired \table -> do
-        void $ PGNotify.watchInsertOrUpdateTable table do
-            sessions <- (get #sessions) <$> readIORef autoRefreshServer
-            sessions
+    subscriptions <-  subscriptionRequired |> mapM (\table -> PGNotify.watchInsertOrUpdateTable table do
+                sessions <- (get #sessions) <$> readIORef autoRefreshServer
+                sessions
                     |> filter (\session -> table `Set.member` (get #tables session))
                     |> map (\session -> get #event session)
                     |> mapM (\event -> MVar.tryPutMVar event ())
-            pure ()
-
-
+                pure ())
+    modifyIORef autoRefreshServer (\s -> s { subscriptions = subscriptions })
```

Comment:
Shouldn't it append those instead of overriding? Otherwise existing subscriptions will get lost.

```suggestion
    modifyIORef autoRefreshServer (\s -> s { subscriptions = get #subscriptions s <> subscriptions })
```